smartsim/_core/entrypoints/file_operations.py

```diff
@@ -0,0 +1,272 @@
+# BSD 2-Clause License
+#
+# Copyright (c) 2021-2024 Hewlett Packard Enterprise
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions and the following disclaimer.
+#
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions and the following disclaimer in the documentation
+#    and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import argparse
+import base64
+import collections
+import os
+import pickle
+import re
+import shutil
+from ast import literal_eval
+from distutils import dir_util  # pylint: disable=deprecated-module
+
+
+def _check_path(file_path: str) -> str:
+    """Given a user provided path-like str, find the actual path to
+        the directory or file and create a full path.
+
+    :param file_path: path to a specific file or directory
+    :raises FileNotFoundError: if file or directory does not exist
+    :return: full path to file or directory
+    """
+    full_path = os.path.abspath(file_path)
+    if os.path.isfile(full_path):
+        return full_path
+    if os.path.isdir(full_path):
+        return full_path
+    raise FileNotFoundError(f"File or Directory {file_path} not found")
+
+
+def move(parsed_args: argparse.Namespace) -> None:
+    """Move a file
+
+    Sample usage:
+        python _core/entrypoints/file_operations.py move
+        /absolute/file/src/path /absolute/file/dest/path
+
+        source path: Path to a source file to be copied
+        dest path: Path to a file to copy the contents from the source file into
+    """
+    _check_path(parsed_args.source)
+    _check_path(parsed_args.dest)
+    shutil.move(parsed_args.source, parsed_args.dest)
+
+
+def remove(parsed_args: argparse.Namespace) -> None:
+    """Write a python script that removes a file when executed.
```

Comment:
What do these mean by "write a python script"?  Are we writing a script?  Same question for ``copy`` method.

Reply:
woops, left this from before. Changing this now